mesonbuild/backend/vs2010backend.py

```diff
@@ -1779,17 +1767,20 @@ def path_normalize_add(path, lis):
         for o in custom_objs:
             additional_objects.append(o)
 
+        # VS automatically links CustomBuild outputs whose name ends in .obj or .res,
+        # but the others need to be included explicitly
+        explicit_link_gen_objs = [obj for obj in gen_objs if not any(obj.endswith(ext) for ext in ['.obj', '.res'])]
```

Comment:
```suggestion
        explicit_link_gen_objs = [obj for obj in gen_objs if not obj.endswith(('.obj', '.res'))]
```

endswith accepts a tuple of possible endings, you don't need to use `any(generator-expression)`.

Reply:
TIL! Done.